velox/functions/prestosql/aggregates/RegisterAggregateFunctions.cpp

```diff
@@ -155,6 +156,7 @@ void registerAllAggregateFunctions(
     bool withCompanionFunctions,
     bool onlyPrestoSignatures,
     bool overwrite) {
+  registerJsonType();
```

Comment:
Hmm, I'm not sure if this is the right practice in Velox - to register a type in a function that registers functions.
Looking at other places - we are already doing it in multiple code sites, so it should be fine.

Reply:
Agree, I first tried registering this in Function Signature.cpp,  but that leads to a circular dependency in resolving libraries. 

We could keep this for now and do a refactor in the future if needed.